schema/validator.go

```diff
@@ -200,6 +202,31 @@ func validateConfig(r io.Reader) error {
 	return nil
 }
 
+func checkArchitecture(Architecture string, Variant string) {
+	validCombins := map[string][]string{
+		"arm":      {"v6", "v7", "v8"},
+		"arm64":    {"v8"},
```

Comment:
Shouldn't these both include the empty string?

Reply:
Fixed.